src/styles.scss

```diff
@@ -1,9 +1,45 @@
 /* You can add global styles to this file, and also import other style files */
 @import 'custom-theme';
+@import "app/styles/main";
+
 
 body {
     overflow-x: hidden;
     background: white;
     color: black;
 }
 
+.certificateClass {
+    .mat-dialog-container {
+      background-color: gold;
```

Comment:
This certificate doesn't look that good, what do you people think @shreyans29 @Raxerz @Buddhiprabha ?

Reply:
@lunayach please make some changes to it and add screenshots either here or on the slack channel

Reply:
It looks good now, but seeing the color gold.. it looks like the PR hasn't been updated.